### ISSUE TITLE
[Bug] about:config shows cryptic error

### ISSUE THREAD

**Issue by opusforlife2** [mozilla-mobile/fenix#9313]
## Steps to reproduce
Try to open about:config.
### Expected behavior
Fenix either opens it or tells you it's not available.
### Actual behavior
Weird URL in address bar and cryptic error shown on page.
URL:
```
resource://android/assets/low_and_medium_risk_error_pages.html?&title=Cannot%20Complete%20Request&button=Try%20Again&description=%3Cp%3EAdditional%20information%20about%20this%20problem%20or%20error%20is%20currently%20unavailable.%3C/p%3E&image=&showSSL=false&badCertAdvanced=Advanced%E2%80%A6&badCertTechInfo=%3Clabel%3ESomeone%20could%20be%20trying%20to%20impersonate%20the%20site%20and%20you%20should%20not%20continue.%3C/label%3E%20%3Cbr%3E%3Cbr%3E%20%3Clabel%3EWeb%20sites%20prove%20their%20identity%20via%20certificates.%20Firefox%20Preview%20does%20not%20trust%20%3Cb%3Eabout:config%3C/b%3E%20because%20its%20certificate%20issuer%20is%20unknown,%20the%20certificate%20is%20self-signed,%20or%20the%20server%20is%20not%20sending%20the%20correct%20intermediate%20certificates.%3C/label%3E&badCertGoBack=Go%20Back%20(Recommended)&badCertAcceptTemporary=Accept%20the%20Risk%20and%20Continue
```

### Device information

* Android device: Xiaomi Mi 5
* Fenix version: 4.1

**Comment by lindongbin** [comment]
It is time to enable about:config on all firefox version.

**Comment by grravity** [comment]
I am curious, can anyone tell me will most of the flags I currently have set in Fennec be able to be used on Fenix? Or is it going to be missing like 90% of the same flags.
One thing I love about Fennec is that it is similar to my build on desktop, and I can easily deal with keeping flags similar and extensions.
The performance of Fennec and bugs on the other hand are poor.

I'm hoping Fenix keeps the customizability for a desktop like experience for users who wish for it, and also provide better performance.

**Comment by B0pol** [comment]
Duplicate of #7865

**Comment by opusforlife2** [comment]
Oh well. Firefox Preview is gone. There is only Firefox Beta and Firefox Nightly now, and both have about:config, so closing this.

**Comment by grravity** [comment]
@opusforlife2 Firefox Preview exists on F-Droid still, is this going to be replaced?

**Comment by Dunexus** [comment]
The F-Droid version is maintained and built by a third party, not by Mozilla itself.

**Comment by opusforlife2** [comment]
@grravity @Dunexus From the readme of the https://gitlab.com/rfc2822/fdroid-firefox repo I think you're both referring to:

> "This unofficial F-Droid repository is hosted using Gitlab pages.
A daily scheduled pipeline
downloads the APKs directly from the app developers and then updates this F-Droid repository (which lives
on Gitlab pages)."

and
> "The APKs are unaltered and hence still signed by the app developers."

**Comment by Dunexus** [comment]
Oh you are right, thank you. I thought it was built by the maintainer of the repo, my bad, sorry !

**Comment by filips123** [comment]
> Oh well. Firefox Preview is gone. There is only Firefox Beta and Firefox Nightly now, and both have about:config, so closing this.

Once Fenix is released as stable, it won't have `about:config` unless something changes. Anyway, this issue should stay closed because it is duplicate of #7865.
